content/academics.html

```diff
@@ -21,35 +35,84 @@ <h3> Attendance </h3>
+</p>
+<div class="half second">
+  <span class="graph-title"> Failures </span>
+  <div id="failing">
+  </div>
+</div>
+
+<div class="vbx2"></div>
+
+<h3>Options</h3>
+<p class="half">
+  <span class="big">
+  2 respondents completed an option.
+  </span>
+  Graduating with an Option can be complicated to do in Engineering. With a restrictive amount of electives,
+  most students require to either overload a term or terms, or complete courses over coop in order to meet
+  all the requirements for the Option and for graduating in SE.
+</p>
+<div class="half second">
+  <span class="graph-title"> Type of Options </span>
+  <div id="options">
+  </div>
+</div>
+
+<div class="vbx2"></div>
+
+<h3>Favourite Professors</h3>
+<p class="half">
+  <span class="big">
+  The most favourite professor with 14% of the votes is Martin Pei.
+  </span>
+  For this section, students were only able to select one professor among all the
+  professors they had during their academic career.
+  <br /><br />
+  Some professors listed in this section did not teach a core course, which means
+  not every student had the opportunity to be taught with said professors.
+</p>
+<div class="half second">
+  <span class="graph-title"> Favourite Professors </span>
+  <div id="prof-cloud">
+  </div>
+</div>
+
+<div class="vbx2"></div>
 
 
 <h2> Courses </h2>
-<h3> Favourite required courses </h3>
+<h3> Favourite courses </h3>
 <p class="half">
 <span class="big">
-  Although 3B was the most difficult term, it had Concurrency (CS343), the most popular course.
+  Among core courses, <a href="https://uwflow.com/course/math239" target="_blank">MATH 239 (Introduction to Combinatorics)</a>
+  and <a href="https://uwflow.com/course/cs349" target="_blank">CS 349 (User Interface)</a> were the most popular courses. Among electives 
+  <a href="https://uwflow.com/course/ece454" target="_blank">ECE 454 (Distributed Computing)</a>
+  was the most popular.
+  <!-- Although 3B was the most difficult term, it had Concurrency (CS343), the most popular course. -->
 </span>
-  The top 8 favourited courses were all between 2A and 3B. Although
-  these terms are slightly more difficult, they also happen to have the most
-  interesting content for students.
-  <br /><br />
-  Students are also required to take several technical electives to fulfill
+  The top favourited courses were all taken between 3A and 4B, this can either be because
+  students don't remember much of their first and second year courses, or that students
+  enjoyed more higher level courses.
+  <!-- Students are also required to take several technical electives to fulfill
   their credits.  The most popular electives included Graphics (CS488),
-  Distributed Computing (ECE 454), and Human-Computer Interaction (CS 449).
+  Distributed Computing (ECE 454), and Human-Computer Interaction (CS 449). -->
 </p>
 <div class="half second">
   <span class="graph-title">Favourite mandatory courses</span>
```

Comment:
i saw that music courses were in the graph; should the title change to "Favourite courses" (instead of mandatory) to reflect that?

Reply:
Yes, I didn't update the title yet because I am still planning to update the graph later. I decided to add in the addenum survey a question about their favourite mandatory course and elective course, it seemed to make more sense to have 2 different graphs than having one about their favourite uni course